app/models/task.py

```diff
@@ -1,5 +1,39 @@
 from app import db
+from datetime import datetime 
+from sqlalchemy import Column, ForeignKey, Integer, String
+from sqlalchemy.orm import relationship
+
 
 
 class Task(db.Model):
-    task_id = db.Column(db.Integer, primary_key=True)
+    task_id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    title = db.Column(db.String)
+    description = db.Column(db.String)
+    completed_at = db.Column(db.DateTime, nullable = True)
+    goal_id = db.Column(db.Integer, db.ForeignKey('goal.goal_id'), nullable=True)
+    
+    goal = db.relationship("Goal", back_populates="tasks")
+
+    def to_result(self):
```

Comment:
Small suggestion, but I think it might be nice to either rename this to `to_dict` to conform with `Goal`, or to try a name like `to_response`.